frontend/packages/topology/src/components/side-bar/TopologyEdgeResourcesPanel.tsx

```diff
@@ -17,6 +18,7 @@ type TopologyEdgeResourcesPanelProps = {
 };
 
 const TopologyEdgeResourcesPanel: React.FC<TopologyEdgeResourcesPanelProps> = ({ edge }) => {
```

Comment:
Missed `Connections` string passed to `SidebarSectionHeading` on line 36.

Reply:
Oooh, missed the title missing. 

With the pseudo lang:
![Screen Shot 2020-12-01 at 11 36 18 AM](https://user-images.githubusercontent.com/8126518/100769023-86539600-33c9-11eb-9ac6-c92bda5592d3.png)

With the "default" lang:
![Screen Shot 2020-12-01 at 11 36 23 AM](https://user-images.githubusercontent.com/8126518/100769027-86539600-33c9-11eb-8027-b15fc52ca8e3.png)